doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -1655,6 +1658,9 @@ def build_payload_istag(payload_tag_name: str, payload_entry: PayloadEntry) -> D
             "from": {
                 "kind": "DockerImage",
                 "name": payload_entry.dest_pullspec,
+            },
+            "importPolicy": {
+                "importMode": "PreserveOriginal"
```

Comment:
I'm assuming this would not have sideffect when the image is not a manifest list